client/signup/steps/import-url/index.jsx

```diff
@@ -2,132 +2,197 @@
+			return null;
+		}
+
+		const { isSiteImportableError, translate } = this.props;
 
-		this.props.setValidationMessage( message );
-	}
+		switch ( isSiteImportableError.code ) {
+			case 1000001:
+				return translate( 'This does not appear to be a valid URL or website address.' );
+			// @TODO differentiate an unreachable site from one that's not compatible.
+			case 1000002:
+				return translate(
+					"We're not able to reach that site at this time, or it's not compatible with our URL importer."
+				);
+		}
+
+		return translate( 'There was an error with the importer, please try again.' );
+	};
 
 	renderContent = () => {
-		const { isInputDisabled, urlInputValidationMessage, urlInputValue, translate } = this.props;
-		const { showValidation } = this.state;
+		const { isLoading, urlInputValue, translate } = this.props;
+		const { showUrlMessage } = this.state;
+		const urlMessage = this.getUrlMessage();
+		const noticeText = translate(
+			"Please wait, we're checking to see if we can import this site."
+		);
 
 		return (
-			<div className="import-url__wrapper">
-				<FormTextInputWithAction
-					placeholder={ translate( 'Please enter the full URL of your site.' ) }
-					action="Continue"
-					onAction={ this.handleAction }
-					label={ translate( 'URL' ) }
-					onChange={ this.handleInputChange }
-					defaultValue={ urlInputValue }
-					disabled={ isInputDisabled }
-					onBlur={ this.handleInputBlur }
-				/>
-				{ showValidation ? (
-					<FormInputValidation
-						text={ urlInputValidationMessage || translate( 'This URL is valid.' ) }
-						isError={ !! urlInputValidationMessage }
+			<Fragment>
+				{ isLoading && (
+					<Notice
+						className="import-url__notice"
+						status="is-info"
+						icon="info"
+						isLoading={ true }
+						text={ noticeText }
+						showDismiss={ false }
+						isCompact={ true }
 					/>
-				) : (
-					<FormSettingExplanation>
-						{ translate( 'Please enter the full URL of your site.' ) }
-					</FormSettingExplanation>
 				) }
-			</div>
+				<Card className="import-url__card" tagName="div">
+					<form className="import-url__form" onSubmit={ this.handleSubmit }>
+						<FormTextInput
+							className="import-url__url-input"
+							placeholder="https://example.wixsite.com/mysite"
+							disabled={ isLoading }
+							defaultValue={ urlInputValue }
+							onChange={ this.handleInputChange }
+							onBlur={ this.handleInputBlur }
+							inputRef={ this.handleInputRef }
+						/>
+						<FormButton
+							className="import-url__submit-button"
+							disabled={ isLoading }
+							busy={ isLoading }
+							type="submit"
+						>
+							{ translate( 'Continue' ) }
+						</FormButton>
+					</form>
+					{ showUrlMessage &&
+						urlMessage && <FormInputValidation text={ urlMessage } isError={ !! urlMessage } /> }
+				</Card>
+
+				<div className="import-url__example">
+					<p className="import-url__example-urls">
+						{ translate( 'Example URLs', {
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?